src/types/ant.ts

```diff
@@ -190,6 +201,14 @@ export interface AoANTRead {
     opts?: AntReadOptions,
   ): Promise<number>;
   getBalances(opts?: AntReadOptions): Promise<Record<WalletAddress, number>>;
+  getHandlers(): Promise<AoANTHandler[]>;
+  validate({
+    validations,
+  }: {
+    validations?: Partial<
+      Record<AoANTHandler, ({ ant }: { ant: AoANTRead }) => Promise<boolean>>
+    >;
+  }): Promise<Record<AoANTHandler, { valid: boolean; error?: string }>>;
```

Comment:
nit: update readme with these handlers and examples

Reply:
https://github.com/ar-io/ar-io-sdk/pull/237/commits/bce76d2c4315ad8ab31e604ac23680c04d7f7141